src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/TextAreaPart.cshtml

```diff
@@ -18,4 +18,4 @@
         }
     }
 }
-<textarea id="@fieldId" name="@fieldName" class="@fieldClass" placeholder="@Model.Value.Placeholder">@fieldValue</textarea>
+<textarea id="@fieldId" name="@fieldName" class="@fieldClass" placeholder="@Model.Value.Placeholder" rows="@(Model.Value.Rows.HasValue ? Model.Value.Rows : null)">@fieldValue</textarea>
```

Comment:
You can't print rows="" as this is an invalid HTMl. The attribute should be printed only if there is a value

Reply:
The `null` will remove the entire attribue

Reply:
It won't. Check the HTML output

Reply:
I tried many times before on different attributes, but I will double-check

Reply:
@MikeAlhayek I did a quick sample and it rid-off the attribute if the value is `null`

Reply:
Okay. I did not know that. But if you confirmed it, I trust you

Reply:
As I told you above I tried that a lot of time in the past, and double check it after your comment to make sure if it's works with textarea